pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -317,10 +317,10 @@ func (plugin *kubenetNetworkPlugin) SetUpPod(namespace string, name string, id k
 	if err != nil {
 		return err
 	}
-	if res.IP4 == nil || res.IP4.IP.String() == "" {
+	if res.IP4 == nil || res.IP4.IP.IP.String() == "" {
```

Comment:
Don't both net.IP and net.IPAddr's `String` print `"<nil>"` (even for technically non-nil structs) for invalid stuff, not `""`? Is this right?